Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN cp ./target/release/syntect_server /syntect_server
 ################################
 FROM golang:1.15.2-alpine@sha256:4d8abd16b03209b30b48f69a2e10347aacf7ce65d8f9f685e8c3e20a512234d9 as hss
 
-RUN apk add --no-cache git=2.26.2-r0
+RUN apk add --no-cache git
```

Comment:
that version of git no longer exists in the package repos. Was there a reason that specific version was pinned?

Reply:
It's good practice to pin dependencies in general, Hadolint enforces this in our main repo but we haven't enabled it here yet. Can you pin the version to whatever does exist?